.evergreen/config.in.yml

```diff
@@ -143,8 +143,8 @@ functions:
         working_dir: src
         binary: bash
         env:
-          AUTH: 'auth'
-          SSL: 'ssl'
+          AUTH: "auth"
```

Comment:
Did you accidentally run prettier on this file or something?

Reply:
Intentionally, I have to flip our settings for quotes to prevent the entire file from turning into a diff

Reply:
If we want to lint this, can we add it to our lint config?  If not, can we revert?  imo there is marginal value in linting this file.  we have no consistency between when we use quotes / no quotes, I don't think a linter can enforce that.

Also, because I don't fully understand - the changes here are only in the PR because you have format on save enabled, right?  None of the substantiative changes in this PR require linting (there are no quotes).  Is there a technical reason you can't disable format on save or only commit the relevant chunks of this file to avoid extra formatting changes in the diff?

Reply:
> can we add it to our lint config?

Since we run prettier through eslint we're limited to JS/TS from that config. But I have added a check, particularly for a CI gen script, we've often merged changes along with a format, we can avoid that going forward.

I don't have this enabled as format on save I ran it manually